crates/inspection-service/src/inspection_client.rs

```diff
@@ -10,6 +10,31 @@ pub struct InspectionClient {
     url: Url,
 }
 
+#[derive(Clone, Debug)]
+pub enum MetricValue {
```

Comment:
Is this an unrelated change ?

Reply:
this is required for this PR.

voting power is u128. We don't have u128 guage. So I have to use f64 guage. When I use f64 guage, this code breaks in a smoke test that checks for when voting power is u128.

Seems like I am the first one using f64 with values above u64::MAX.